pkg/gardenlet/operation/botanist/vpnseedserver.go

```diff
@@ -39,6 +39,7 @@ func (b *Botanist) DefaultVPNSeedServer() (vpnseedserver.Interface, error) {
 		HighAvailabilityNumberOfSeedServers:  b.Shoot.VPNHighAvailabilityNumberOfSeedServers,
 		HighAvailabilityNumberOfShootClients: b.Shoot.VPNHighAvailabilityNumberOfShootClients,
 		VPAUpdateDisabled:                    b.Shoot.VPNVPAUpdateDisabled,
+		SeedPodNetworkV4:                     b.Seed.GetInfo().Spec.Networks.Pods,
```

Comment:
How does this work for an IPv6-only single-stack seed?

Reply:
This is somewhat of a misnomer. The network is passed over to VPN regardless. The IPv4 vs. IPv6 check happens only on the VPN side. If a IPv6 pod range is provided, it is used on the vpn client wrt routing back to the seed and it is ignored on the server wrt mapping to the 241/8 range. It'll work either way, hence I am changing the name back to `SeedPodNetwork` to avoid such confusion.